app/(auth)/sign-up/page.jsx

```
@@ -171,89 +173,70 @@
-                        <md-icon slot="leading-icon">person</md-icon>
-                    </md-outlined-text-field>
+                <md-outlined-text-field
+                    error={state.errors.name}
+                    required={true}
+                    className='max-w-[560px] w-[80vw]'
+                    label="Tên của bạn"
+                    value={state.formData.name}
+                    onInput={handleInputChange('name')}
+                    autoFocus={true}
+                >
+                    <md-icon slot="leading-icon">person</md-icon>
```

Biome (1.9.4)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)